pkg/patterns/declarative/reconciler.go

```diff
@@ -134,6 +136,40 @@ func (r *Reconciler) reconcileExists(ctx context.Context, name types.NamespacedN
 	if err != nil {
 		return reconcile.Result{}, err
 	}
+
+	// dynamic config
+	dynamicClientset, err := dynamic.NewForConfig(r.config)
+	if err != nil {
+		log.Error(err,"Unable to create dynamic client")
+		return reconcile.Result{}, err
+	}
+
+	newItems := []*manifest.Object{}
+	for _, obj := range objects.Items {
+
+		// Uses unsafe method?? Is it safe?
+		getOptions := metav1.GetOptions{}
+		gvk, _ := meta.UnsafeGuessKindToResource(obj.GroupVersionKind())
+		ns := obj.UnstructuredObject().GetNamespace()
+		unstruct, err := dynamicClientset.Resource(gvk).Namespace(ns).Get(context.Background(),
+			obj.Name, getOptions)
+		if err != nil{
+			log.WithValues("name", obj.Name).Error(err, "Unable to get resource")
+		}
+		if unstruct != nil {
+			annotations := unstruct.GetAnnotations()
+			if ignoreAnnotation, ok := annotations["addons.operators.ignore"]; ok {
+				if ignoreAnnotation == "true" {
```

Comment:
We can just use an empty value for the annotation i.e. just check `if ok`.  yaml makes values like "true" annoyingly difficult to set...